lib/daru/index.rb

```diff
@@ -330,5 +341,11 @@ def values
     def inspect
       "Daru::MultiIndex:#{self.object_id} (levels: #{levels}\nlabels: #{labels})"
     end
+
+    def factor_out
+      # Remove levels not needed for display
+      useless = levels.index { |l| l.length > 1 }
+      drop_left_level useless
+    end
```

Comment:
And about showing the `:one`; if I remember correctly, I had ported that functionality from Python's pandas. I think they follow the same convention if there's only one element with top level index `:a`.